charts/tempo-distributed/README.md

```diff
@@ -225,16 +236,18 @@ The memcached default args are removed and should be provided manually. The sett
 | memcached.extraEnv | list | `[]` | Environment variables to add to memcached pods |
 | memcached.extraEnvFrom | list | `[]` | Environment variables from secrets or configmaps to add to memcached pods |
 | memcached.host | string | `"memcached"` |  |
+| memcached.image.pullPolicy | string | `"IfNotPresent"` | Memcached Docker image pull policy |
+| memcached.image.registry | string | `nil` | The Docker registry for the Memcached image. Overrides `global.image.registry` |
+| memcached.image.repository | string | `"memcached"` | Memcached Docker image repository |
+| memcached.image.tag | string | `"1.5.17-alpine"` | Memcached Docker image tag |
 | memcached.podAnnotations | object | `{}` | Annotations for memcached pods |
 | memcached.podLabels | object | `{}` | Labels for memcached pods |
-| memcached.pullPolicy | string | `"IfNotPresent"` | Memcached Docker image pull policy |
 | memcached.replicas | int | `1` |  |
-| memcached.repository | string | `"memcached"` | Memcached Docker image repository |
 | memcached.resources | object | `{}` | Resource requests and limits for memcached |
 | memcached.service | string | `"memcached-client"` |  |
-| memcached.tag | string | `"1.5.17-alpine"` | Memcached Docker image tag |
 | memcachedExporter.enabled | bool | `false` | Specifies whether the Memcached Exporter should be enabled |
 | memcachedExporter.image.pullPolicy | string | `"IfNotPresent"` | Memcached Exporter Docker image pull policy |
+| memcachedExporter.image.registry | string | `nil` | The Docker registry for the Memcached Exporter image. Overrides `global.image.registry` |
```

Comment:
should this also be updated along with the memcached image?

Reply:
Each service could use a different registry or uses the same registry that is set under the `global.image` key.